src/VSCodeExtension/README.md

```diff
@@ -1,12 +1,10 @@
-# Microsoft Quantum Development Kit Preview
+# Microsoft Quantum Development Kit Preview for .NET6
```

Comment:
is the right space .NET6 or .NET 6?

Reply:
It looks like it should be .NET 6 https://dotnet.microsoft.com/en-us/download/dotnet/6.0

Reply:
I'll update it accordingly.